test-tap/assert.js

```diff
@@ -1608,22 +1674,22 @@ test('.notThrowsAsync()', gather(t => {
 	});
 }));
 
-test('.notThrowsAsync() returns undefined for a fulfilled promise', t => {
+test('.notThrowsAsync() returns true for a fulfilled promise', t => {
```

Comment:
Same as https://github.com/avajs/ava/pull/2601/files#r504349454, this is a change in method contract, of which I'm unsure of the larger implications.